aptos-move/framework/aptos-framework/sources/vesting_without_staking.spec.move

```diff
@@ -94,8 +102,13 @@ spec aptos_framework::vesting_without_staking {
         pragma aborts_if_is_partial = true;
         include AdminAborts;
         let vesting_contract = borrow_global<VestingContract>(contract_address);
-        ensures !simple_map::spec_contains_key(vesting_contract.shareholders, shareholder_address);
-        ensures !simple_map::spec_contains_key(vesting_contract.beneficiaries, simple_map::spec_get(vesting_contract.beneficiaries, shareholder_address));
+        let post vesting_contract_post = borrow_global<VestingContract>(contract_address);
```

Comment:
Do not use `borrow_global`, use spec version `global` instead.

Reply:
Updated!